src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -71,6 +110,7 @@ public void mercury_search_verifySearchLayout() {
     Assertion.assertTrue(search.isInputFieldSearchIconVisible());
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
   @Test(groups = "mercury_search_userIsRedirectedToSearchResultsPage")
   public void mercury_search_userIsRedirectedToSearchResultsPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)